### PR TITLE
Deliver build configuration

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Add text file with information about SDL build configuration as part of SDL delivery.

### CODE REVIEW

**Comment by LuxoftAKutsan** [comment]
@theresalech please review

File: proposals/nnnn-deliver-build-configuration.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- start.sh
+- libappenders.so        
+- log4cxx.properties  
+- plugins     
+- smartDeviceLinkCore       
+
+
+Only these files (except 3rd party libraries) are required for SDL testing.
+But there is no information about flags that were used for SDL compiling. 
+And there is no way to find out it only from delivered files.
```
Comment (theresalech) [inline proposals/nnnn-deliver-build-configuration.md]: Can you please re-word?  I believe this should say "And there is no way to find out this information from delivered files."

File: proposals/nnnn-deliver-build-configuration.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ - EXTENDED_POLICY (regulates policy flow)
+ - REMOTE_CONTROL (switch on/off remote control functionality)
+ - BUILD_BT_SUPPORT (switch on/off Bluetooth support)
+ - BUILD_USB_SUPPORT (switch on/off USB support)
+ - ENABLE_SECURITY (switch on/off security support)
+ - EXTENDED_MEDIA_MODE (switch on/off additional media features)
+ - TELEMETRY_MONITOR (share information about CPU/MEM usage and time consumptions for RPC processing) 
+ - HMI (used hmi type: Dbus or web HMI)
+ 
+Some of this flags are not supported, but exist, and may affect SDL behavior.
```
Comment (theresalech) [inline proposals/nnnn-deliver-build-configuration.md]: Please change "this" to "these"

Comment (LuxoftAKutsan) [reply proposals/nnnn-deliver-build-configuration.md]: @theresalech fixed

File: proposals/nnnn-deliver-build-configuration.md (hunk 1 above)
Comment (LuxoftAKutsan) [reply proposals/nnnn-deliver-build-configuration.md]: @theresalech fixed

**Pull request merged**